components/LessonsChallenges.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import _ from 'lodash'
+import '../scss/lessonsChallenges.scss'
+
+type ChallengeStatusProps = {
+  challengesData: Challenge[]
+}
+
+type LessonChallengeProps = {
+  lessons: LessonChallenge[]
+}
+
+type LessonChallenge = {
+  order: number
+  title: string
+  totalChallenges: number
+  correctAnswers: number
+  challenges: Challenge[]
+}
+
+type Challenge = {
+  challengeNumber: number
+  challengeStatus: string
+}
+
+export const ChallengeStatus: React.FC<ChallengeStatusProps> = props => {
+  const displayChallengeStatus = props.challengesData.map(eachChallenge => {
+    let challengeStatus = 'default_challenge_status'
+    if (eachChallenge.challengeStatus === 'passed') {
+      challengeStatus = 'passed_challenge_status'
+    }
+    if (eachChallenge.challengeStatus === 'wrong answer') {
```

Comment:
we should stick with using one word, `wrong`

Reply:
understood